.github/workflows/ci.yml

```
@@ -97,26 +97,20 @@ jobs:
     name: Check pre-commit
     runs-on: ubuntu-latest
     needs: build_docker_container
+    container:
+      image: docker.pkg.github.com/${OWNER}/${DOCKER_CACHE_REPO}/${DOCKER_TAG}
+      credentials:
+        username: ${{ CI_USER }}
```

this probably won't work either, but even if it did, it would have to be 
```suggestion
        username: ${{ env.CI_USER }}
```